minifier/minify.py

```diff
@@ -296,6 +297,8 @@ def filter_tags(tokens):
 
 def rename(tokens):
     replacements = dict({
+        # Constants
```

Comment:
There's actually already a section at the bottom of the replacements on L459 for constants. Please move them up here to go with your new addition. Sorry the replacements are a mess, the hard coding is a pain but it will soon be replaced with a better version. Thanks.